src/iter.rs

```diff
@@ -16,8 +16,8 @@ pub struct MiniconfIter<M: ?Sized, const L: usize, const TS: usize> {
 
     /// The remaining length of the iterator.
     ///
-    /// It is used to provide `Iterator::size_hint()`
-    /// and may be None to indicate unknown length.
+    /// It is used to provide an exact and trusted `Iterator::size_hint()`.
```

Comment:
I'm not sure what "trusted" is supposed to mean. I don't really think it's needed to say that it's correct...

Reply:
It is important information as this goes beyond the size_hint promises.
It would be `TrustedLen` if that was stable.